plugins/defaultplugins/aclplugin/vppdump/dump_vppcalls.go

```diff
@@ -262,3 +528,18 @@ func getIcmpMatchRule(r acl_api.ACLRule) *acl.AccessLists_Acl_Rule_Matches_IpRul
 	}
 	return &icmp
 }
+
+// decodeIPv4Address converts first four elements of provided byte array to IPv4 address
+// as a string.
+func decodeIPv4Address(addr []byte) string {
```

Comment:
Is this function necessary? Doesn't it do the same as `net.IP(addr).To4().String()`?

Reply:
it is possible, I check

Reply:
`net.IP` is basically `type IP []byte`